platform_versioned_docs/version-23.3.0/enterprise/advanced-topics/use-iam-role.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Use AWS IAM roles"
```

Comment:
Arg. I commented on 23.1.0 since it was the first file presented. 

My concerns about Data Explorer and Data Studio are obvioiusly only relevant to the appropriate Tower versions that support it but the rest holds true.